src/StopwatchPlugin.php

```diff
@@ -39,11 +40,11 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
         $this->stopwatch->start($eventName, self::CATEGORY);
 
         return $next($request)->then(function (ResponseInterface $response) use ($eventName) {
-            $this->stopwatch->stop($eventName, self::CATEGORY);
+            $event = $this->stopwatch->stop($eventName);
 
-            return $response;
+            return $response->withHeader(self::HEADER, $event->getDuration());
```

Comment:
i wonder if there is a more elegant way than modifying the response with an additional header.